releases/release-1.8/README.md

```diff
@@ -0,0 +1,43 @@
+# Kubeflow 1.8
+
+### Links
+
+- [Release Team](release-team.md)
+- [Calendar](https://arrik.to/kf-release-team-cal)
+- [Meeting Notes](https://arrik.to/kf-release-team-notes)
+- [Recordings](https://arrik.to/kf-release-team-recordings)
+- [Retrospective Document](https://bit.ly/kf-release-1-6-retro)
+- Project Board
+- Contact
+  - [#release](https://app.slack.com/client/T7QLHSH6U/C9V2WT2KV) on slack
+  - [@release-team](https://github.com/orgs/kubeflow/teams/release-team) on GitHub
+
+### TL;DR
+
+The 1.8 release cycle is proposed as follows
+
+- **Friday, May 5th 2023**     : Week 0      - Release team is formed
+- **Monday, May 8th 2023**     : Week 1  C\* - Release cycle begins
+- **Friday, May 12th 2023**    : Week 1  C\* - Roadmaps are finalized
+- **Wednesday, Aug 16th 2023** : Week 15 C\* - Feature freeze
+- **Wednesday, Aug 21st 2023** : Week 16     - Manifests testing and bug fixing starts
+- **Friday, Sept 1st 2023**    : Week 17 C\* - Manifests testing and bug fixing ends
+- **Monday, Sept 4th 2023**    : Week 18     - Docs update starts
+- **Monday, Sept 4th 2023**    : Week 18     - Distribution testing starts
+- **Friday, Sept 22nd 2023**   : Week 20     - Distribution testing ends
+- **Monday, Sept 25th 2023**   : Week 21 C\* - Pre-release bug fixing stage starts
+- **Friday, Oct 6th 2023**     : Week 22     - Pre-release bug fixing stage ends
```

Comment:
Does the timeline capture testing the fix? or does this mean, this is the last day to get a fix in?

Reply:
For this stage I was thinking of last minute bugs that could be found by the distributions, WG and/or the CI. So during these weeks I'd expect WG to merge fixes in kubeflow/manifests that will be tested on the CI and then we cut a RC.